### ISSUE TITLE
Add /ping endpoint for compatibility with telegraf/influxdb endpoints

### ISSUE THREAD

**Issue by DaveWK** [open-telemetry/opentelemetry-collector-contrib#29594]
### Component(s)

receiver/influxdb

### Is your feature request related to a problem? Please describe.

Some third party apps utilize a /ping endpoint which is available on telegraf/influxdb and will bail out on sending data when it gets an unexpected ping response (currently for the otel-contrib influxdb receiver it returns 404)

### Describe the solution you'd like

Add the `/ping`to the opentelemetry plugin as an endpoint and have it return 204.
An example curl of telegraf:
```
curl -vv localhost:8086/ping
*   Trying 127.0.0.1:8086...
* Connected to localhost (127.0.0.1) port 8086 (#0)
> GET /ping HTTP/1.1
> Host: localhost:8086
> User-Agent: curl/8.0.1
> Accept: */*
>
< HTTP/1.1 204 No Content
< X-Influxdb-Version: 1.0
< Date: Thu, 30 Nov 2023 21:16:10 GMT
<
* Connection #0 to host localhost left intact
```

### Describe alternatives you've considered

I tried setting a proxy in front of the endpoint to catch the /ping and return the 204 but it's convoluted to add and support a whole proxy service in an attempt to emulate the native behaviour of a the native telegraf/influxdb rpc

**Comment by jacobmarble** [comment]
@jve-engineering LGTM!

**Comment by crobert-1** [comment]
_Removing the `needs triage` label as the code owner supports the idea._

**Comment by mx-psi** [comment]
I believe this was fixed by #29662
